docker-compose.yml

```diff
@@ -20,14 +20,14 @@ services:
      - "7687:7687"
      - "6479:6479"
      - "7474:7474"
-  elasticsearch_dev:
-    image: docker.elastic.co/elasticsearch/elasticsearch:6.5.4
-    environment:
-      - discovery.type=single-node
-      - ES_JAVA_OPTS=-Xms512m -Xmx512m
-    ports:
-      - "9300:9300"
-      - "9200:9200"
+#  elasticsearch_dev:
```

Comment:
Please remove the comment, we need this for the correct setup.
If you don't want to start the elasticsearch_dev container just do something like
```
docker-compose up -d
docker-compose stop elasticsearch_dev
``` 
instead of changing the `docker-compose.yml`